docs/CREATOR_GUIDE.md

```
@@ -0,0 +1,1023 @@
+
+#### 3. Create a New Branch
+
+```bash
+git checkout -b add-your-quickstart-name
+```
+
+#### 4. Add Your Resource
+
+Each learning resource lives in its own folder under `docs/quickstarts/`:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)